IMC_R/Modeling_Jeff/Intro to Logistic Regression.md

```diff
@@ -0,0 +1,99 @@
+# Intro to Logistic Regeression Modeling
+
+## What is Logistic Regression
+
+Hello there. Have you ever wondered how people were able to classify a large group of items into 2 categoires? Well this is the perfect article for you! The technique that is able to accomplish such a goal is called Logistic Regression. Now what is Logistic Regression, you may ask? Logistic Regression is a classification technique that determines what values in our data set belong within a binary group. You have seen certain examples of this throughout your life such as gender clasification (Boy/Girl), posotive and negative terminology (good/bad), and even when it comes to taking exams (pass/fail). This is a very important procedure to understand because you'll be able to correctly classify what groups does your data belong in to reduce any form of error.
```

Comment:
i would also introduce why we are using logistic rather than linear. because the predicted value is discrete and binary either 0 or 1, not 1.2,1..5, or 1,6783 etc